src/Core/Models/Imports/TetraCubeCreature.cs

```diff
@@ -0,0 +1,106 @@
+﻿using System.Text.Json.Serialization;
+
+namespace DMPowerTools.Core.Models.Imports
+{
+    public class TetraCubeCreature
+    {
+        public int Id { get; set; }
+        public string Name { get; set; }
+        public string Size { get; set; }
+        public string Type { get; set; }
+        public string Tag { get; set; }
+        public string Alignment { get; set; }
+        public int HitDice { get; set; }
+        public string ArmorName { get; set; }
+        public int ShieldBonus { get; set; }
+        public int NatArmorBonus { get; set; }
+        public string OtherArmorDesc { get; set; }
+        public int Speed { get; set; }
+        public int BurrowSpeed { get; set; }
+        public int ClimbSpeed { get; set; }
+        public int FlySpeed { get; set; }
+        public bool Hover { get; set; }
+        public int SwimSpeed { get; set; }
+        public bool CustomHP { get; set; }
+        public bool CustomSpeed { get; set; }
+        public string HpText { get; set; }
+        public string SpeedDesc { get; set; }
+        public int StrPoints { get; set; }
+        public int DexPoints { get; set; }
+        public int DexModifier
```

Comment:
This should probably live in somewhere else as it’s the calculation for 5e in general.